Lib/test/test_ast.py

```diff
@@ -965,6 +965,12 @@ def test_literal_eval_complex(self):
         self.assertRaises(ValueError, ast.literal_eval, '3+(0+6j)')
         self.assertRaises(ValueError, ast.literal_eval, '-(3+6j)')
 
+    def test_literal_eval_malformed(self):
```

Comment:
I find this name is too generic. Maybe `test_literal_eval_malformed_dict_nodes` ?